app/assets/stylesheets/custom.scss

```diff
@@ -2,3 +2,12 @@
 //
 // You can also create SCSS files in the app/assets/stylesheets/custom/
 // folder and they will be loaded automatically.
+
+.top-bar {
+  .top-bar-title a img {
+    @include breakpoint(medium) {
+      width: 7rem;
```

Comment:
Why this line? :thinking:

Be the way, if there's a reason to keep it, how about moving it to line 56 in `app/assets/stylesheets/custom/layout.scss`, since we've already got rules for this selector there?

Reply:
With this rule, the logo is almost the same size as the previous one we had. Without this rule, the new logo and navigation menu are too big, IMO.